README.md

```diff
@@ -13,3 +16,14 @@ Interestingly enough though, the goal of the project has slowly evolved over tim
 
 ## Lua
 sm64coopdx is moddable via Lua, similar to Roblox and Garry's Mod's Lua APIs. To get started, click [here](docs/lua/lua.md) to see the Lua documentation.
+
+## Compilation
+You will only need to compile the game if you want to set [build flags](https://github.com/coop-deluxe/sm64coopdx/wiki/Build-Flags).
+### 32bit 
+For 32bit compilation you will need to apend `TARGET_BITS 32` so you can compile it in 32bits. There is no guarantee that it will work/work correctly. 
+
+## Android port
+There is currently no android port.
+
+### Discord 
+[Public Invite](https://discord.gg/TJVKHS4)
```

Comment:
use this link instead: https://discord.gg/eXhpSRhCyM
thats the official permanent link

Reply:
Changed